clang/include/clang/Driver/Options.td

```diff
@@ -2548,6 +2548,14 @@ def fsanitize_trap_EQ : CommaJoined<["-"], "fsanitize-trap=">, Group<f_clang_Gro
 def fno_sanitize_trap_EQ : CommaJoined<["-"], "fno-sanitize-trap=">, Group<f_clang_Group>,
                            Visibility<[ClangOption, CLOption]>,
                            HelpText<"Disable trapping for specified sanitizers">;
+def fsanitize_nonmerged_handlers_EQ
+    : CommaJoined<["-"], "fsanitize-nonmerged-handlers=">,
+      Group<f_clang_Group>,
+      HelpText<"Enable non-merged handlers for specified sanitizers">;
```

Comment:
Allow compiler to merged handlers

Reply:
There are non-_EQ version for trap and recover, we probably want them for consistency

Reply:
Done